### DISCUSSION TITLE
Endless reconciliation loop

### DISCUSSION THREAD

**Discussion by repjak** [strimzi/strimzi-kafka-operator#10591] [Q&A] [answered]
Hello!

After performing some operations on the Kafka cluster (re-creating the cluster, deleting topics with kafka-topics.sh, mm2 setup etc.) I started to observe cpu-throttling of strimzi pod (at 1 CPU request).

I found out strimzi's logs contain a pattern of messages like:
```
2024-09-16 15:44:06 INFO  AbstractOperator:510 - Reconciliation #2888(watch) Kafka(myns/mykafka): Kafka mykafka in namespace myns was MODIFIED
2024-09-16 15:44:16 INFO  AbstractOperator:536 - Reconciliation #2888(watch) Kafka(myns/mykafka): reconciled
```

repeating ca. every 10 seconds.

This seems like the reconciliation fails silently in an endless loop. Or is it normal?

Attaching Kafka CRD and the debug log.

[strimzi_recon.log](https://github.com/user-attachments/files/17016280/strimzi_recon.log)
[mykafka.yaml.txt](https://github.com/user-attachments/files/17016283/mykafka.yaml.txt)

Any advice would be appreciated.

Edit:

Strimzi version: 0.43.0
K8s version: 1.30.2-4 (OVH Managed k8s service)

**Comment by scholzj** [comment, marked as answer]
That looks like it might be the same issue as here? Unupdated CRD resources? https://github.com/strimzi/strimzi-kafka-operator/issues/10576

**Reply by repjak** [reply]
That was it. I forgot that the helm chart does not upgrade CRDs. Thanks!
